stable_radical_optimization/stable_radical_opt.py

```diff
@@ -0,0 +1,334 @@
+import argparse
+import pathlib
+import logging
+import time
+from typing import Tuple
+
+import sqlalchemy
+from sqlalchemy import create_engine
+
+import numpy as np
+import pandas as pd
+
+import rdkit
+from rdkit import Chem
+from rdkit.Chem import Mol, MolToSmiles
+
+from rlmolecule.molecule.policy import preprocessor
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+#sys.path.append('/projects/rlmolecule/pstjohn/models/20201031_bde/')
+#from preprocess_inputs import preprocessor as bde_preprocessor
+#bde_preprocessor.from_json('/projects/rlmolecule/pstjohn/models/20201031_bde/preprocessor.json')
+# TODO would we need a different preprocessor.json file here
+# than we have in rlmolecule.molecule.policy.preprocessor.json?
+bde_preprocessor = preprocessor.load_preprocessor()
```

Comment:
Yeah, so the BDE model was trained on a different set of data and we need to keep the preprocessor consistent, so we'd want to make sure we can load that bde-specific preprocessor from the bde model directory.